src/gui/qgsprojectionselectionwidget.h

```diff
@@ -100,6 +101,46 @@ class GUI_EXPORT QgsProjectionSelectionWidget : public QWidget
      */
     static QString crsOptionText( const QgsCoordinateReferenceSystem &crs ) SIP_SKIP;
 
+    /**
+     * Returns TRUE if the widget will show a warning to users when they select a CRS which has
+     * low accuracy.
+     *
+     * \see setShowAccuracyWarnings()
+     * \since QGIS 3.20
+     */
+    bool showAccuracyWarnings() const;
+
+    /**
+     * Sets whether the widget will \a show warnings to users when they select a CRS which has
+     * low accuracy.
+     *
+     * \see showAccuracyWarnings()
+     * \since QGIS 3.20
+     */
+    void setShowAccuracyWarnings( bool show );
+
+    /**
+     * Sets the original source \a ensemble datum name.
+     *
+     * If set, CRS accuracy warnings will not be shown when the selected CRS in the widget has a matching
+     * ensemble datum, regardless of the ensemble's accuracy.
+     *
+     * \see sourceEnsemble()
+     * \since QGIS 3.20
+     */
+    void setSourceEnsemble( const QString &ensemble );
+
+    /**
+     * Returns the original source ensemble datum name.
+     *
+     * If set, CRS accuracy warnings will not be shown when the selected CRS in the widget has a matching
+     * ensemble datum, regardless of the ensemble's accuracy.
+     *
+     * \see setSourceEnsemble()
+     * \since QGIS 3.20
+     */
+    QString sourceEnsemble() const;
```

Comment:
does that need to be public, or exist at all ? It doesn't seem there are users of that method.

Reply:
Not strictly, but I think it's good practice to have getters for all properties we can set. I've been hurt in the past when writing plugins targeting older qgis versions and there's missing getters -- it can be a real roadblock for plugin development.